client/components/NewMeetup.js

```diff
@@ -2,20 +2,164 @@ import React, {Component} from 'react';
 import { connect } from 'react-redux';
 import {  } from '../store';
 import { Redirect } from 'react-router-dom';
+import * as _ from 'lodash';
+import axios from 'axios';
+import {daysInMonth} from '../../utils';
 
 class NewMeetup extends Component {
-    constructor(){
+    constructor(props){
         super();
+        let { user } = props,
+            now = new Date(),
```

Comment:
Do you need to store each of these values in a variable this way or is there some way to grab everything in one variable and unpack as needed?

Reply:
just need now_year, and now_month. Going to take out now_date, and now_hour.
It's more for readability.